src/test/java/io/github/microcks/testcontainers/MicrocksContainerTest.java

```diff
@@ -37,11 +38,7 @@
 
 import java.io.File;
 import java.time.Duration;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```

Comment:
Could you please restore the explicit import list?

Reply:
Done